src/js/tabs/setup.js

```diff
@@ -369,29 +370,39 @@ setup.initialize = function (callback) {
 
             if (supported && ispConnected()) {
                 const buildRoot = `https://build.betaflight.com/api/builds/${FC.CONFIG.buildKey}`;
+
                 const buildConfig = `<span class="buildInfoBtn" title="${i18n.getMessage(
                     "initialSetupInfoBuildConfig",
                 )}: ${buildRoot}/json">
-                                    <a href="${buildRoot}/json" target="_blank"><strong>${i18n.getMessage(
+                    <a href="${buildRoot}/json" target="_blank"><strong>${i18n.getMessage(
     "initialSetupInfoBuildConfig",
 )}</strong></a></span>`;
+
                 const buildLog = `<span class="buildInfoBtn" title="${i18n.getMessage(
                     "initialSetupInfoBuildLog",
                 )}: ${buildRoot}/log">
-                                    <a href="${buildRoot}/log" target="_blank"><strong>${i18n.getMessage(
+                    <a href="${buildRoot}/log" target="_blank"><strong>${i18n.getMessage(
     "initialSetupInfoBuildLog",
 )}</strong></a></span>`;
-                build_info_e.html(`<span class="buildInfoBtn" title="${i18n.getMessage(
```

Comment:
Maybe controversial, these 6 lines APPEAR to be left over from a refactor that moved the Options button into its own function. Note that the result of `.html` here is immediately replaced by line 390/388. I can leave this, but I think it's a NOOP and the real version of this starts at line 411/422.